src/parameterList.ts

```diff
@@ -1,45 +1,43 @@
+  customZimTags: 'ZIM tags metadata (semi-colon separated)',
+  customZimLanguage: 'ZIM ISO639-3 content language code',
   customMainPage: 'Allow to configure a custom page as welcome page.',
-  filenamePrefix: 'For the part of the ZIM filename which is before the format & date parts.',
+  filenamePrefix: 'Part of the ZIM filename which is before the format & date parts.',
   format:
-    'Specify a flavour for the scraping. If missing, scrape all article contents. Each --format argument will cause a new local file to be created but options can be combined. Supported options are:\n * novid: no video & audio content\n * nopic: no pictures (implies "novid")\n * nopdf: no PDF files\n * nodet: only the first/head paragraph (implies "novid")\nFormat names can also be aliased using a ":"\nExample: "... --format=nopic:mini --format=novid,nopdf"',
+    'Specify a flavour for the scraping. If missing, scrape all article contents. Each --format argument will cause a new local file to be created but options can be combined. Supported options are:\n * novid: no video & audio content\n * nopic: no pictures (implies "novid")\n * nopdf: no PDF files\n * nodet: only the first/head paragraph (implies "novid")\nFlavour can be named (and corresponding ZIM metadata will be created) using a ":":\nExample: "--format=nopic,nodet:mini"',
   keepEmptyParagraphs: 'Keep all paragraphs, even empty ones.',
-  mwWikiPath: 'Mediawiki wiki base path (per default "/wiki/")',
-  mwActionApiPath: 'Mediawiki API path (per default "/w/api.php")',
-  mwRestApiPath: 'Wikimedia Rest API path (per default "/api/rest_v1")',
-  mwMediaWikiRestApiPath: 'Mediawiki Rest API path (per default "w/rest.php/v1/page/")',
-  mwModulePath: 'Mediawiki module load path (per default "/w/load.php")',
-  mwDomain: 'Mediawiki user domain (thought for private wikis)',
-  mwUsername: 'Mediawiki username (thought for private wikis)',
-  mwPassword: 'Mediawiki user password (thought for private wikis)',
+  mwWikiPath: 'MediaWiki wiki base path (by default "/wiki/")',
+  mwActionApiPath: 'MediaWiki API path (by default "/w/api.php")',
+  mwRestApiPath: 'Wikimedia REST API path (by default "/api/rest_v1")',
+  mwMediaWikiRestApiPath: 'MediaWiki REST API path (by default "w/rest.php/v1/page/")',
+  mwModulePath: 'MediaWiki module load path (by default "/w/load.php")',
+  mwDomain: 'MediaWiki user domain (thought for private wikis)',
+  mwUsername: 'MediaWiki username (thought for private wikis)',
+  mwPassword: 'MediaWiki user password (thought for private wikis)',
   minifyHtml: 'Try to reduce the size of the HTML',
   outputDirectory: 'Directory to write the downloaded content',
-  publisher: "ZIM publisher meta data, per default 'Kiwix'",
+  publisher: "ZIM publisher meta data, by default 'Kiwix'",
   redis: 'Redis path (redis:// URL or path to UNIX socket)',
   requestTimeout: 'Request timeout - in seconds(default value is 120 seconds)',
```

Comment:
Request timeout - in seconds(default value is 120 seconds)

=>

Request timeout in seconds (default is 120s)

Reply:
Done